### ISSUE TITLE
openDevTools ignores options

### ISSUE THREAD

**Issue by eriedl-kiban** [electron/electron#7503]
* Electron version: 1.4.2
* Operating system: Windows (have not tried it yet on Mac and Linux)

`webContents.openDevTools([options])` ignores any options given and defaults to either the last dock state or, if there are no user preferences yet, it will default to the right.

**Comment by eriedl-kiban** [comment]
The issue still persists in 1.4.4. According to the change log it was fixed with this release. The only `mode` option that has any effect on how the developer tools window is opened is `detach`. Every other `mode` option is still opens the developer tools attached to the right.

**Comment by deepak1556** [comment]
I was able to reproduce the issue on `v1.4.4` but not on `master`. Seems like the release doesn't contain the fix.

```
> git ls-tree v1.4.4 vendor/brightray                                                                                                           
160000 commit 8b074eca1217db6621553a3f7764c907ef617547	vendor/brightray

```

/cc @kevinsawicki

**Comment by kevinsawicki** [comment]
> I was able to reproduce the issue on v1.4.4 but not on master. Seems like the release doesn't contain the fix.

Yeah, apologies for this, the brightray submodule wasn't bumped after the fix there was merged so it didn't make it in 1.4.4, will be in 1.4.5 though (and is on master).

**Comment by elantion** [comment]
I still got this problem in version 1.7.6. Have any special parameters or setting required to support this options?
